extensions/openapi-generator/deployment/src/main/java/io/quarkus/openapi/generator/deployment/template/QuteTemplatingEngineAdapter.java

```diff
@@ -0,0 +1,74 @@
+package io.quarkus.openapi.generator.deployment.template;
+
+import java.io.IOException;
+import java.util.Map;
+
+import io.quarkus.qute.Engine;
+import io.quarkus.qute.ReflectionValueResolver;
+import io.quarkus.qute.Template;
+import org.openapitools.codegen.api.AbstractTemplatingEngineAdapter;
+import org.openapitools.codegen.api.TemplatingExecutor;
+
+public class QuteTemplatingEngineAdapter extends AbstractTemplatingEngineAdapter {
+
+    public static final String IDENTIFIER = "qute";
+    public static final String[] INCLUDE_TEMPLATES = {
+            "additionalEnumTypeAnnotations.qute",
+            "additionalModelTypeAnnotations.qute",
+            "beanValidation.qute",
+            "beanValidationCore.qute",
+            "beanValidationHeaderParams.qute",
+            "bodyParams.qute",
+            "enumClass.qute",
+            "enumOuterClass.qute",
+            "formParams.qute",
+            "headerParams.qute",
+            "pathParams.qute",
+            "pojo.qute",
+            "queryParams.qute"
+    };
+    public final Engine engine;
+
+    public QuteTemplatingEngineAdapter() {
+        // TODO: inject from qute buildStep
```

Comment:
When is this class actually used? Qute does not provide a fully initiliazed `Engine` instance during the build phase. But I think it's OK to use a standalone `Engine` instance for this extension only.

Reply:
Oh, this class is instantiated by the OpenAPI Generator tool via service loader. So, are you saying that I can't inject a Qute Engine during the build phase? I guess initializing it manually will be the default option then. Correct?

Reply:
Yes.